src/startupscriptgenerator/src/common/logging.go

```diff
@@ -6,36 +6,36 @@
 package common
 
 import (
+	"common/consts"
 	"fmt"
 	"os"
-	"startupscriptgenerator/common/consts"
 	"strings"
 	"time"
 
-	"github.com/Microsoft/ApplicationInsights-Go/appinsights"
-	"github.com/Microsoft/ApplicationInsights-Go/appinsights/contracts"
+	"github.com/microsoft/ApplicationInsights-Go/appinsights"
+	"github.com/microsoft/ApplicationInsights-Go/appinsights/contracts"
 )
 
-const SHUTDOWN_CLOSE_TIMEOUT time.Duration = 3 * time.Second
-const SHUTDOWN_EXIT_TIMEOUT time.Duration = 6 * time.Second
+const ShutDownCloseTimeOut time.Duration = 3 * time.Second
+const ShutDownExitTimeOut time.Duration = 6 * time.Second
 
 type Logger struct {
 	AiClient    appinsights.TelemetryClient
 	LoggerName  string
 	AppName     string
-	OperationId string
+	OperationID string
 }
 
 // Represents unique identifier that can be used to correlate messages with the build logs
-var buildOpId string
+var buildOpID string
 
-func SetGlobalOperationId(appRootPath string) {
+func SetGlobalOperationID(appRootPath string) {
```

Comment:
please move these styling changes to a separate PR to make this one smaller and more focused on the re-org

Reply:
I think this is fine for this PR since for me at this point I want to make sure that the development experience is good and that I did not do anything wrong, but are we not doing these kind of changes? ;-) let's all agree to do these kind of changes in separate PRs instead